scripts/dictionaries.py

```diff
@@ -0,0 +1,40 @@
+#Dictionaries are key value mapping datastructure
+
+d={'a':"A",'b':"B"}
+print(d)#{'a': 'A', 'b': 'B'}
+
+#Get values from d
+print(d['a'])#A
+
+#add element in d
```

Comment:
what are the other ways to add element in the dict ?

Reply:
fixed
i.e. d.update({'d':"D"})